testsuite/src/test/java/org/ironjacamar/eis/support/EchoConnection.java

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
We shouldn't have a dedicated resource adapter for this, as we won't use it for other things.

Also, all test resource adapters needs to go under the .rars package